bin/server.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
+var cors = require('cors');
 var app = express();
+app.use(cors());
```

Comment:
aha, I see what you mean now - sorry I didn't see the body-parser code below, I thought you were adding something new - please make this change and we'll merge